pkg/image/installer_image_test.go

```diff
@@ -251,12 +250,9 @@ func TestTarInstallerUnsetKSOptions(t *testing.T) {
 	img.Platform = testPlatform
 
 	mfs := instantiateAndSerialize(t, img, mockPackageSets(), nil, nil)
-	// the tar installer doesn't set a custom kickstart path unless the
-	// unattended option is enabled, so the inst.ks option isn't set and
-	// interactive-defaults.ks is used
+
 	assert.Contains(t, mfs, fmt.Sprintf(`"inst.stage2=hd:LABEL=%s"`, isolabel))
-	assert.Contains(t, mfs, fmt.Sprintf("%q", osbuild.KickstartPathInteractiveDefaults))
-	assert.NotContains(t, mfs, "osbuild.ks") // no mention of the default (custom) value
+	assert.Contains(t, mfs, fmt.Sprintf(`"inst.ks=hd:LABEL=%s:/osbuild.ks"`, isolabel))
```

Comment:
Nitpick: Shouldn't the unit test consider `img.Kickstart.Path` value? :thinking: